### DISCUSSION TITLE
does this require wal_level=logical set up?

### DISCUSSION THREAD

**Discussion by Cariaga** [shortishly/pgec#10] [Q&A] [answered]
does this require wal_level=logical set up?
can it work in wal_level=replica?

**Comment by shortishly** [comment, marked as answer]
We require `wal_level=logical`, so that we can replicate just the tables that we need rather than the whole database. Without `wal_level=logical` you'll get the following error when creating a publication for your tables like:

WARNING:  wal_level is insufficient to publish logical changes
HINT:  Set wal_level to "logical" before creating subscriptions.
